_includes/tutorials/ksql-concatenation-columns/ksql/markup/answer/concatenation-answer.adoc

```diff
@@ -0,0 +1,7 @@
+Select the fields you want to combine and use the `+` operator to concatenate them together into one field:
```

Comment:
```suggestion
Select the fields you want to combine and use the `+` operator to concatenate them into one field:
```

It is my understanding that concatenate doesn't need the word `together` (few instances of this usage in the KT)